extension/src/debug.ts

```diff
@@ -125,31 +117,22 @@ class LispLaunchConfigurationProvider implements vscode.DebugConfigurationProvid
     private _server?: Net.Server;
 
     async resolveDebugConfiguration(folder: vscode.WorkspaceFolder | undefined, config: vscode.DebugConfiguration, token?: vscode.CancellationToken): Promise<vscode.DebugConfiguration> {
-        // if launch.json is missing or empty
-        if (need2AddDefaultConfig(config)) {
-            config.type = launchCfgType;
-            config.name = launchCfgName;
-            config.request = 'launch';
-        }
+
+        var newConfig = {} as vscode.DebugConfiguration;
+        newConfig.type = launchCfgType;
+        newConfig.name = launchCfgName;
+        newConfig.request = 'launch';
 
         if (vscode.window.activeTextEditor)
-            config.program = vscode.window.activeTextEditor.document.fileName;
+             newConfig.program = vscode.window.activeTextEditor.document.fileName;
 
-        if (config["type"] === launchCfgType) {
+        if (newConfig["type"] === launchCfgType) {
             // 1. get acad and adapter path
-            //2. get acadRoot path
-            //2.1 get acadRoot path from launch.json
-
+            // 2. get acadRoot path
             let productPath = "";
```

Comment:
Can we just:
`let productPath = getExtensionSettingString(LAUNCH_PROC);`
If this API returns "", then there should be no difference.